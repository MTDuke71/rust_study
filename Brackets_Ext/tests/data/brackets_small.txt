()
[]{}()
([{}])
a(b[c]{d}e)f 123 ðŸš€
([)]
(((
abc
]abc
{[()(){}([])]}
([{}])x{y[z]}
(]
Ã©)
