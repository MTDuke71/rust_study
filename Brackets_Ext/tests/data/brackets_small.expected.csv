line,ok,index,kind,expected,found,open_index
1,true,,,,,
2,true,,,,,
3,true,,,,,
4,true,,,,,
5,false,2,MismatchedPair,],),
6,false,2,UnclosedOpenings,),,2
7,true,,,,,
8,false,0,UnexpectedClosing,,],
9,true,,,,,
10,true,,,,,
11,false,1,MismatchedPair,),],
12,false,2,UnexpectedClosing,,),
